qemu/tests/cfg/device_assignment.cfg

```diff
@@ -1,13 +1,26 @@
 - device_assignment:
     only pf_assignable
     virt_test_type = qemu
+    pre_command = "modprobe vfio-pci"
```

Comment:
we need to check interrupt remapping and vfio-pci module for all device assignment  test, no matter pf or vf.  Only one pre_command  to add vfio-pci  module is not enough.